DiscordBot.Modules/CommandModules/CodeSnippetModule.cs

```diff
@@ -1,37 +1,36 @@
 ﻿using Discord.Commands;
 
 using DiscordBot.Domain;
-using DiscordBot.Domain.Configuration;
+using DiscordBot.Domain.Database;
 
-using Microsoft.Extensions.Options;
-
-using System.Collections.Generic;
-using System.IO;
+using System;
 using System.Linq;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
-using Newtonsoft.Json;
 
 namespace DiscordBot.Modules.CommandModules
 {
     [Group("code")]
-    [Summary("Saves and shows code snippets.")]
+    [Summary("Speichert und zeigt Code-Snippets an.")]
```

Comment:
Nono

Reply:
But this is a German bot 😀 So, not really German, but its language is German. For example take a look at GrillBot. It is also a Discord bot written in C#. Do you understand anything: https://github.com/Misha12/GrillBot/blob/master/Grillbot/Modules/AdminModule.cs#L32 ?